pkg/kubelet/kubelet.go

```diff
@@ -957,7 +959,17 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 	}
 
 	// Start volume manager
-	go kl.volumeManager.Run(wait.NeverStop)
+	readyCh := make(chan bool, 1)
+	go kl.volumeManager.Run(readyCh, wait.NeverStop)
```

Comment:
Why did we decide to have this channel be created explicitly? Comment?
@jingxu97 @saad-ali